server/models/tests/testHelpers.ts

```diff
@@ -153,13 +152,24 @@ export async function makeValidComment(
   authorID: string,
   postID: string
 ): Promise<Comment> {
-  return makeComment(
+  return await makeComment(
     'This is the body of the comment!\
     This is the body of the comment!',
     authorID,
     postID
   );
 }
+export async function getAllTags(): Promise<Tag[]> {
+  return await TagModel.findAll();
+}
+
+export async function getPostTags(
+  postId: string
+): Promise<(Tag & PostTag)[] | undefined> {
+  const post = await PostModel.findByPk(postId);
+
+  return await post?.getTags();
```

Comment:
couldn't post be null if no postId is found? then you would call getTags() on null

Reply:
its just for tests, but this looks like it was from the Tags PR.